### ISSUE TITLE
Bug: nodejs/node and other big repositories missing information when adding a new dashboard item

### ISSUE THREAD

**Issue by 0-vortex** [open-sauced/open-sauced#949]
### Describe the bug

When trying to get information about some big repositories like [nodejs/node](https://github.com/nodejs/node) the gql data we receive in chunks contains missing information.

### To Reproduce

1. Log network activity through console
1. Go to the open-sauced dashboard and try to add [nodejs/node](https://github.com/nodejs/node) to your watch list.
1. Check the network tab
1. All the qgl queries return `data=null`

### Expected behavior

The repository should be added.

### Screenshots

Sample JSON of the responses when querying:

```json
{
    "data": {
        "gitHub": {
            "viewer": {
                "repository": {
                    "id": "MDEwOlJlcG9zaXRvcnkzNDkxMjU2MTE=",
                    "data": null,
                    "issues": {
                        "totalCount": 1,
                        "nodes": [{
                            "id": "MDU6SXNzdWU4NjMyMzc3MzE=",
                            "full_name": "open-sauced/open-sauced",
                            "body": "",
                            "number": 1,
                            "labels": {
                                "nodes": []
                            }
                        }]
                    }
                }
            }
        }
    }
}
```

```json
{
    "data": {
        "__typename": "Mutation",
        "gitHub": {
            "createIssue": {
                "issue": {
                    "id": "MDU6SXNzdWU4NzEyOTY2NzA=",
                    "title": "nodejs/node"
                }
            }
        }
    }
}
```

```json
{
    "data": {
        "gitHub": {
            "viewer": {
                "repository": {
                    "id": "MDEwOlJlcG9zaXRvcnkzNDkxMjU2MTE=",
                    "data": null,
                    "issues": {
                        "totalCount": 2,
                        "nodes": [{
                            "id": "MDU6SXNzdWU4NzEyOTY2NzA=",
                            "full_name": "nodejs/node",
                            "body": "",
                            "number": 2,
                            "labels": {
                                "nodes": []
                            }
                        }, {
                            "id": "MDU6SXNzdWU4NjMyMzc3MzE=",
                            "full_name": "open-sauced/open-sauced",
                            "body": "",
                            "number": 1,
                            "labels": {
                                "nodes": []
                            }
                        }]
                    }
                }
            }
        }
    }
}
```

```json
{
    "data": {
        "gitHub": {
            "viewer": {
                "repository": {
                    "id": "MDEwOlJlcG9zaXRvcnkzNDkxMjU2MTE=",
                    "data": null,
                    "issues": {
                        "totalCount": 2,
                        "nodes": [{
                            "id": "MDU6SXNzdWU4NzEyOTY2NzA=",
                            "full_name": "nodejs/node",
                            "body": "",
                            "number": 2,
                            "labels": {
                                "nodes": []
                            }
                        }, {
                            "id": "MDU6SXNzdWU4NjMyMzc3MzE=",
                            "full_name": "open-sauced/open-sauced",
                            "body": "",
                            "number": 1,
                            "labels": {
                                "nodes": []
                            }
                        }]
                    }
                }
            }
        }
    }
}
```

### Desktop

- OS: all
- Browser: all
- Version: all

### Smartphone

- Device: all
- OS: all
- Browser: all
- Version: all

### Additional context

https://clips.twitch.tv/SpotlessSpunkyBadgerGingerPower-3T3nD5AGuUbjRhCy

**Comment by filiptronicek** [comment]
Yep, this is known, tried this a couple of months ago with Linux, but I really don't know what the backend issue is if there is an execution limit or something.

![example of a non-working repo (torvalds/linux)](https://user-images.githubusercontent.com/29888641/116611244-6d2e2180-a936-11eb-9da1-40cb83a953d5.png)


@bdougie are there any logs that we keep? Does OneGraph have any?

**Comment by 0-vortex** [comment]
Most repositories with more than 20.000 history items are somewhat limited as far as I am aware from other applications but I don't know details. Will try to further investigate after getting the local OneGraph set up and documented properly.

**Comment by 0-vortex** [comment]
@bdougie shadow fixed this recently ? :D

**Comment by filiptronicek** [comment]
Quoting from https://github.com/open-sauced/open-sauced/issues/1065#issuecomment-873555619:

> This leads me to wonder, what exactly it is about Linux that makes the query fail. The number of commits is notable, but AFAIK there is no point in which we fetch any of the commits (or do we?), but maybe it has something to do with the contributors because there's more than 5,000 contributors, and the REST API returns this:
```json
{
  "message": "The history or contributor list is too large to list contributors for this repository via the API.",
  "documentation_url": "https://docs.github.com/rest/reference/repos#list-repository-contributors"
}

```

**Comment by 0-vortex** [comment]
> Quoting from [#1065 (comment)](https://github.com/open-sauced/open-sauced/issues/1065#issuecomment-873555619):
> 
> > This leads me to wonder, what exactly it is about Linux that makes the query fail. The number of commits is notable, but AFAIK there is no point in which we fetch any of the commits (or do we?), but maybe it has something to do with the contributors because there's more than 5,000 contributors, and the REST API returns this:
> 
> ```json
> {
>   "message": "The history or contributor list is too large to list contributors for this repository via the API.",
>   "documentation_url": "https://docs.github.com/rest/reference/repos#list-repository-contributors"
> }
> ```

Can confirm with OneGraph backend response:

```json
{
    "errors": [
        {
            "message": "Error retrieving contributors for this repository",
            "path": [
                "gitHub",
                "repositoryOwner",
                "repository",
                "contributors_oneGraph"
            ]
        }
    ],
    "data": {
        "gitHub": {
            "repositoryOwner": {
                "repository": null
            }
        }
    }
}
```

I think the solution here is to modify the query to limit contributors display to the first 10 or the most recently active users. If this is the only blocker it would auto-fix #1065 for now, as we do not have any other way to "break" a repository outside of contributors count.
